ticc/Ticc.php

```diff
@@ -124,6 +131,15 @@ private function run_deploy() {
          * Deploy a set of changes - a target change and all
          * dependencies of it, or all un-deployed changes.
          */
+        if($this->masterplan->minus($this->deployedplan)->plan_empty_p()) {
```

Comment:
Arguably the top level itself is a Change, and there does not need to be any plan,json or DDL scripts for a Change to be valid... Of course, valid is different from useful. It might be good to warn if the masterplan is empty.

However, the way this is written, I think it would:
1. not throw the error in an empty top level because there would still be one top level change;
2. throw the error when all changes have been deployed, no matter how many there are.